dev/tasks/verify-rc/github.win.yml

```diff
@@ -29,6 +29,7 @@ jobs:
       {{ key }}: {{ value }}
     {% endfor %}
     {% endif %}
+    timeout-minutes: 60
```

Comment:
I am not against setting the timeout, any specific reason to add it? Those jobs only run on  verifying binaries at the moment and it would be unfortunate to start hitting the timeout there

Reply:
Some jobs such as https://github.com/ursacomputing/crossbow/actions/runs/11393132955/job/31700762815 got stuck when I was working on this. I canceled them manually because we don't have the timeout.

I think that we should have a reasonable timeout to avoid wasting CI time (even when we don't pay money for it). If "60" isn't enough, we can use more long time such as "120". It's better than the default "360".